### DISCUSSION TITLE
Welcome to incubator Discussions!

### DISCUSSION THREAD

**Discussion by sumanashrestha** [moest-np/incubator#2] [Announcements]
## 👋 Welcome!
Baselined requirements and context will be in project specific folder and discussions will be hosted here.

**Comment by 9851173819** [comment]
I am a Secondary level English language teacher in Dhunche Rasuwa, I want to transfer to my hometown Dhading but SMC is not ready to send me there in my place. Previously I missed 3 chances to go to my district. What can be next step ??
